explainer.html

```diff
@@ -225,6 +226,16 @@ <h2>Defining an RDF Dataset Hash and/or Signing RDF Datasets</h2>
     </div>
   </section>
 
+  <section id="out-of-scope">
+    <h2>Out-of-scope Issues for the Working Group Scope</h2>
```

Comment:
```suggestion
    <h2>Out-of-scope Issues for the Working Group</h2>
```
it feels like one "scope" too many...